kafka-quarkus-dev/README.adoc

```diff
@@ -148,13 +148,13 @@ The Quarkus example application in this quick start has three application-scoped
 * You've created the `prices` example Kafka topic.
 
 .Procedure
-. On the command line, navigate to the `guides/quarkus` directory that you imported and enter the following commands to run the Quarkus example application.
+. On the command line, navigate to the `code-examples/quarkus/kafka-quickstart` directory that you imported and enter the following commands to run the Quarkus example application.
 +
 .Running the Quarkus example application
 [source]
 ----
-$ cd ~/guides/quarkus
-$ ./mvnw quarkus:dev
+$ cd ./code-examples/quarkus/kafka-quickstart
+$ ./mvnw quarkus:dev -Dquarkus.profile=dev-env
```

Comment:
As above, just use the dev profile here.

Reply:
Ok. I want it to make SBO dev profile default but it makes sense.